requirements.txt

```diff
@@ -1,11 +1,59 @@
--i https://pypi.org/simple
-certifi==2020.6.20
-chardet==3.0.4
-click==7.1.2; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4'
-colorama==0.4.3
-idna==2.10; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
-requests==2.24.0
-shellingham==1.3.2
-toolz==0.10.0
-typer[all]==0.3.1
-urllib3==1.25.10; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4' and python_version < '4'
+aiofiles==0.7.0; python_version >= "3.6" and python_version < "4.0" \
+    --hash=sha256:c67a6823b5f23fcab0a2595a289cec7d8c863ffcb4322fb8cd6b90400aedfdbc \
+    --hash=sha256:a1c4fc9b2ff81568c83e21392a82f344ea9d23da906e4f6a52662764545e19d4
+async-generator==1.10; python_version < "3.7" and python_version >= "3.6" \
```

Comment:
one doubt...in the README we have `python 3.7+` as requirement and here for some of the dependencies we are setting `python_version < "3.7"` , won't this create an issue on a python 3.7+ environment?

Reply:
That's a great point. The cli client will not work with python < 3.7.1. 

Is there a tool that will stop working if the 3.6 environments are upgraded to 3.7.1+?